buildmaster/ATLAS_2JET_7TEV_R06/filter.py

```diff
@@ -0,0 +1,329 @@
+# Filter for ATLAS_2JET_7TEV_R06
+"""
+Created on Mar  2023
+
+@author: Mark N. Costantini
+"""
+
+import yaml
+import numpy as np
+import pandas as pd
+from scipy.linalg import block_diag, cholesky
+
+
+from validphys.covmats import dataset_inputs_covmat_from_systematics
+from validphys.loader import Loader
+import warnings
+warnings.filterwarnings('ignore')
+
+
+
+def filter_ATLAS_2JET_7TEV_R06(scenario=0,kin_data=True):
```

Comment:
It might be a good idea (for readability, and perhaps reusability) to break this down into several functions, for example by slitting offs the bits that read data, process the kinematics and compute the covmats.

Reply:
Also I feel the kindata processing could as well be a separate function altogether. The boolean flag to only enable it on the first pass feels a bit unnatural.